### PR TITLE
Introduce page header

### PR DESCRIPTION
**Pull request by gokerlek**
"pageheader" folders created. day1 day2 and day3 have new headers.

### CODE REVIEW

File: src/components/pageHeader/pageHeader.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import './pageHeader.css'
```
Comment (onerzafer) [inline src/components/pageHeader/pageHeader.js:1]: naming convention for files is kebabcase so page-header.css and page-header.js

File: src/pages/day1.js [hunk 2]
```diff
@@ -1,63 +1,61 @@
 import { Link } from 'react-router-dom';
 import { Code } from '../components/code/code'
-
+import { PageHeader } from '../components/pageHeader/pageHeader'
 export const Day1 = () => {
     return <div className="day1">
-
-        <div>
-            <p>I have installed NVM by running the following command on the terminal
+        <PageHeader>LOGIN TO REACT</PageHeader>
```
Comment (onerzafer) [inline src/pages/day1.js:6]: As a title "React: very first steps" would be more meaningful

File: src/pages/day2.js [hunk 3]
```diff
@@ -1,8 +1,9 @@
 import { Link } from "react-router-dom"
 import { Code } from '../components/code/code'
-
+import { PageHeader } from '../components/pageHeader/pageHeader'
 export const Day2 = () => {
     return <div className="day2">
+        <PageHeader>CREATING THE BASIC BUILDING STONES OF THE INTERNET SITE</PageHeader>
```
Comment (onerzafer) [inline src/pages/day2.js:6]: "CREATING THE BASIC BUILDING STONES OF THE INTERNET SITE" -> CREATING THE BASIC BUILDING STONES OF A REACT WEBSITE

File: src/components/pageHeader/pageHeader.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+export const PageHeader = ({ children }) => {
+    return <div className="pageHeader">
```
Comment (onerzafer) [inline src/components/pageHeader/pageHeader.js:3]: tag can be h1 instead of div

**Review by onerzafer — CHANGES_REQUESTED** [review]

File: src/components/page-header/page-header.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import './page-header.css'
```
Comment (onerzafer) [inline src/components/page-header/page-header.js:1]: Where is this file?

Comment (gokerlek) [reply src/components/page-header/page-header.js]: file deleted. it is not necessary

**Review by onerzafer — APPROVED** [review]

**Pull request merged**